apps/server/src/app.ts

```diff
@@ -283,7 +257,6 @@ export const shutdown = async (exitCode = 0) => {
 
   expressServer?.close();
   runtimeService.shutdown();
```

Comment:
do we not still need the osc shutdown here?

Reply:
yes we do, good catch